po/de.po

```diff
@@ -608,126 +606,125 @@ msgid "Closed on"
 msgstr "Geschlossen am"
 
 #: GTG/gtk/data/task_editor.ui:376
-#, fuzzy
 msgid "Mark the task as done"
-msgstr "Erledigt"
+msgstr "Aufgabe als erledigt markieren"
 
 #: GTG/gtk/data/task_editor.ui:389
-#, fuzzy
 msgid "Mark the task as to be done"
-msgstr "Unerledigt"
+msgstr "Aufgabe als noch nicht erledigt markieren"
 
 #: GTG/gtk/data/task_editor.ui:402
-#, fuzzy
 msgid "Task action menu"
-msgstr ""
-"\n"
-"Aufgabeninhalt:\n"
-"%s"
+msgstr "Aufgabenaktionen"
 
 #: GTG/gtk/data/task_editor.ui:625
 msgid "Search"
-msgstr "Suchen"
+msgstr "Suche"
 
 #: GTG/plugins/export/export.ui:8
-#, fuzzy
 msgid "Export tasks"
-msgstr "Schlagwörter importieren"
+msgstr "Aufgaben expotieren"
 
 #: GTG/plugins/export/export.ui:38
 msgid "<b>Which tasks do you want to export?</b>"
-msgstr ""
+msgstr "<b>Welche Aufgaben möchten Sie exportieren?</b>"
 
+# TODO: What exactly is being meant here with "browser"? Current taskview?
 #: GTG/plugins/export/export.ui:54
 msgid "Active tasks shown in the browser"
-msgstr ""
+msgstr "Aktive Aufgaben, die gerade in der Aufgabenansicht angezeigt werden"
 
+# TODO: What exactly is being meant here with "browser"? Current taskview?
 #: GTG/plugins/export/export.ui:70
 msgid "Completed tasks shown in the browser"
 msgstr ""
+"Fertiggestellte Aufgaben, die gerade in der Aufgabenansicht angezeigt werden"
 
 #: GTG/plugins/export/export.ui:85
 msgid "Tasks finished last week"
-msgstr ""
+msgstr "Aufgaben, die seit letzter Woche erledigt worden sind"
 
 #: GTG/plugins/export/export.ui:124
 msgid "<b>Select a format for your tasks</b>"
-msgstr ""
+msgstr "<b>Wählen Sie einen Format für die Aufgaben aus</b>"
+
+#: GTG/plugins/export/export.ui:207
+msgid "Open"
+msgstr "Öffnen"
 
 #: GTG/plugins/export/export.ui:221
 msgid "Save"
-msgstr ""
+msgstr "Speichern"
 
 #: GTG/plugins/export/export.ui:263
 msgid "How do you want to access the export plugin?"
-msgstr ""
+msgstr "Wie möchten Sie auf das Exportplugin zugreifen?"
 
 #: GTG/plugins/export/export.ui:278
 msgid "Put an entry in the Plugins menu"
-msgstr ""
+msgstr "Einen Eintrag im Erweiterungsmenü"
```

Comment:
I used that text because of the context: (here I used your translation just as a test, remove the "hinzufügen" for the current string)

![grafik](https://user-images.githubusercontent.com/1196130/92261031-196b5580-eed9-11ea-8fc4-5a2d73847cb4.png)

Reply:
Nevermind then. I wasn't sure where this string was being used.